### ISSUE TITLE
Disabled NcFormBoxSwitch has insufficient contrast for accessibility

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud-libraries/nextcloud-vue#8144]
The contrast between the light blue background and gray text isn't enough to be accessible 

<img width="482" height="63" alt="Image" src="https://github.com/user-attachments/assets/68d066d5-de8d-4784-b7c2-1386c9a24e3f" />

**Comment by ShGKme** [comment]
They don't have to be contrast since they are disabled. We have the same for many interactive elements, and this even was certified in this design.

Ref: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

> Inactive User Interface Components
> 
> User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements. An inactive user interface component is visible but not currently operable. An example would be a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed.
